src/partials/pagination.html

```diff
@@ -0,0 +1,24 @@
+<div class="container not-found">
+      <h2 class="not-found__header">We haven’t found news from<br> this category</h2>
+      <picture>
+        <source
+        media="(min-width: 1280px)"
+        srcset="./images/desk_notfound/image_1.jpg 1x,
+        ./images/desk_notfound/image_1@2x.jpg 2x,
+        ./images/desk_notfound/image_1@3x.jpg 3x"
+      />
+      <source
+      media="(min-width: 768px)"
+      srcset="./images/tablet_notfound/image_2.jpg 1x,
+       ./images/tablet_notfound/image_2@2x.jpg 2x,
+       ./images/tablet_notfound/image_2@3x.jpg 3x"
+      />
+        <source
+          media="(max-width: 767px)"
+          srcset="./images/mob_notfound/image_1.jpg 1x,
+            ./images/mob_notfound/image_1@2x.jpg 2x,
+            ./images/mob_notfound/image_1@3x.jpg 3x"
+        />
+        <img src="./images/desk_notfound/image_1.jpg" alt="Not Found Picture" class="not-found__image"/>
+      </picture>
+</div>
```

Comment:
relocate to template folder, create function renderNotFound() which return this markup